Replace resize_dim() with set_sizes_and_strides()

pcpLiu opened the pull request:
Summary: Replace resize_dim() with set_sizes_and_strides() in   `THTensor_(squeeze1d) in aten/src/TH/generic/THTensor.cpp`

Differential Revision: D14088697

ezyang commented:
CI failures are spurious. Unfortunately the fix hasn't landed yet. Later, you should try rebasing and reexporting.

Commit:
Replace resize_dim() with set_sizes_and_strides() (#17127)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/17127

Replace resize_dim() with set_sizes_and_strides() in   `THTensor_(squeeze1d) in aten/src/TH/generic/THTensor.cpp`

Reviewed By: ezyang

Differential Revision: D14088697

fbshipit-source-id: fa6a36bef2b3e826788a06fda06b8a13cab60f6f

pcpLiu commented:
@ezyang  Is the testing looks good now?

ezyang commented:
Yes LGTM.
